packages/gestalt/src/Datapoint.js

```diff
@@ -0,0 +1,77 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import Text from './Text.js';
+import Heading from './Heading.js';
+import Flex from './Flex.js';
+import IconButton from './IconButton.js';
+import Tooltip from './Tooltip.js';
+import DatapointTrend from './DatapointTrend.js';
+
+type TrendObject = {|
+  accessibilityLabel: string,
+  value: number,
+|};
+
+type Props = {|
+  title: string,
+  value: string,
+  trend?: TrendObject,
+  trendSignal?: 'good' | 'bad' | 'neutral' | 'auto',
+  infoText?: string,
+  size?: 'md' | 'lg',
+|};
+
+export default function Datapoint({
+  title,
+  value,
+  trend,
+  infoText,
+  trendSignal = 'auto',
+  size = 'md',
+}: Props): Node {
+  const infoTextNode = infoText ? (
```

Comment:
This logic can be inlined below, since it's not reused multiple times

(and this can also just be a truthy check)

Reply:
Done!